### PR TITLE
Change Elasticsearch DB URL to point to the database in the COOL DNS account

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description

This pull request changes the Elasticsearch DB URL to point to the database in the COOL DNS account.

## 💭 Motivation and Context

The dmarc-import Elasticsearch database now resides in the COOL DNS account, so that is where we should go to retrieve the data for the Trustworthy Email reports.

## 🧪 Testing

These changes have been applied to production and used to generate this week's Trustworthy Email reports.

## 🚥 Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
⚙️ 🍰

**Review by mcdonnnj — APPROVED** [review]
👍

File: report/generate_trustymail_report.py [hunk 1]
```diff
@@ -73,7 +73,7 @@
 BOD1801_DMARC_RUA_URI = 'mailto:reports@dmarc.cyber.dhs.gov'
 ES_REGION = 'us-east-1'
 ES_URL = 'https://search-dmarc-import-elasticsearch-' \
-    'dtbgkfx23yppmjmothuy6t7wd4.us-east-1.es.amazonaws.com' \
+    'ekc3pdnqzcuifgu4qssctvq4v4.us-east-1.es.amazonaws.com' \
```
Comment (felddy) [inline report/generate_trustymail_report.py:76]: Future variable?

**Review by felddy — APPROVED** [review]
I'm dabbing over here. 🕺

**Pull request merged**